openwisp_controller/config/static/config/js/device.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+django.jQuery(function ($) {
+  if ($(".add-form").length || !$("#device_form").length) {
+    return;
+  }
+
+  // replaces the management ip field with text with the option to edit it
+  var ip_input = $(".field-management_ip > div > input");
+  var initial_ip = ip_input.prop("value");
+  ip_input.after(function () {
+    ip_input.hide();
+    return (
+      '<span class="readonly" id="management_ip_text">' +
+      (initial_ip === "" ? "-n/a-" : initial_ip) +
```

Comment:
Instead of "-n/a-", can we keep it as "-" ? That's how it used to render before this change.